server/aesdsocket.c

```diff
@@ -0,0 +1,287 @@
+#include <stdlib.h>
```

Comment:
@bjornhnelson I see you've already fixed this but make sure to keep binaries out of your repo.